lexicons/app/bsky/feed/post.json

```diff
@@ -38,6 +38,10 @@
             "type": "union",
             "refs": ["com.atproto.label.defs#selfLabels"]
           },
+          "tags": {
+            "type": "array",
+            "items": { "type": "string", "maxLength": 640, "maxGraphemes": 64 }
```

Comment:
include a brief description, eg: "Additional non-inline hashtags describing this post". The description should clarify whether the `#` prefix should be included in these strings or not (i'm not sure). both here and for the in-line (facet) string field, should be consistent about expectation.

at some point we might want a string format like "slug" or "keyword" which has some generic character restrictions. that's a larger protocol tangent and we don't have that today and it should not block this work.

I assume we *will* apply some restrictions server-side when indexing, beyond just length, like "trim whitespace". we could update the description with those details later.

Reply:
Ah good call on description, will add 👍 

As for adding an additional format and other restrictions, @pfrazee and I talked in circles about this last night and we ended up back at the starting point: restrict length/graphemes, but otherwise allow anything.

We _will_ be restrictive with detection of facets in the rich text editor, but when adding additional tags non-inline you'd be able to include things like spaces, etc that would otherwise gum up the rich text editor.

Users may do some wacky stuff, which might make searching for a tag a little difficult, but otherwise handling the tag strings doesn't seem like it should present serious difficulties.

Thoughts?